docs/src/openapi-v1.yaml

```diff
@@ -4259,11 +4259,26 @@ components:
           format: unsignedInteger
         claimedRewards:
           type: array
-          description: Array of eras for which the stakers behind a validator have
-            claimed rewards. Only updated for _validators._
+          description: Array of objects, each containing an `era` and its corresponding `status`,
+            which represents the rewards status of the stakers backing a validator. Only updated
+            for _validators._ This array is populated with values from `stakingLedger.legacyClaimedRewards`
+            or `stakingLedger.claimedRewards`, as well as the `query.staking.claimedRewards` call, depending
+            on whether the queried block is before or after the migration. For more details on the implementation
+            and the migration, refer to the related PR and linked issue.
```

Comment:
Where can the user find this info here?

Reply:
Added specific links of the PR and linked issue. Let me know if this covers your question. I also updated the schema description since I no longer return `lastReward`. Changes included in this [commit](https://github.com/paritytech/substrate-api-sidecar/pull/1445/commits/171e10a45749cfd1c73fddbe297efc8599d77ecb).